src/EFCore.PG/Extensions/NpgsqlDbContextOptionsBuilderExtensions.cs

```diff
@@ -66,16 +98,39 @@ public static class NpgsqlDbContextOptionsExtensions
             var extension = (NpgsqlOptionsExtension)GetOrCreateExtension(optionsBuilder).WithConnection(connection);
             ((IDbContextOptionsBuilderInfrastructure)optionsBuilder).AddOrUpdateExtension(extension);
 
+            ConfigureWarnings(optionsBuilder);
+
             npgsqlOptionsAction?.Invoke(new NpgsqlDbContextOptionsBuilder(optionsBuilder));
```

Comment:
These lines are repeated multiple times, so it's worth to make a private method for them.

Reply:
You're right. But since this class is basically a copy-paste from EF Core, I prefer to just leave things like they are upstream... And there's not huge value here in the private method.